package.json

```diff
@@ -92,17 +92,13 @@
   },
   "lint-staged": {
     "*.{ts,js,css,mjs}": [
-      "prettier --write",
```

Comment:
pulling in [change](https://github.com/ministryofjustice/hmpps-template-typescript/pull/609) from template project that removes prettier config.

Prettier will still be installed as a transitive dependency and that version will use by the generate api types